### PR TITLE
Add TMOTOR VELOXF7V2 target

### PR DESCRIPTION
**Pull request by dengdizzy**
Add TMOTOR VELOXF7V2 target,we TMOTOR test the FC,and FC works well.

### CODE REVIEW

**Comment by haslinghuis** [comment]
Please decide on IMU hardware. Two drivers would be enough as we can use version for other combo's.

**Comment by haslinghuis** [comment]
Same for BARO and MAG

**Comment by dengdizzy** [comment]
Yeah,i also delet baro and mag

**Review by haslinghuis — APPROVED** [review]

**Comment by sugaarK** [comment]
@haslinghuis  not sure what the error is all about!  And @dengdizzy  please squash the commits

**Review by blckmn — DISMISSED** [review]

**Comment by blckmn** [comment]
ERROR: File configs/default/TMTR-TMOTORVELOXF7V2.config has got invalid (DOS) line endings.

**Comment by KarateBrot** [comment]
> haslinghuis not sure what the error is all about! And dengdizzy please squash the commits

@sugaarK Now "squash and merge" is activated by default, so even 100 commits in the PR should be fine 👍

**Comment by dengdizzy** [comment]
> 

Got it,later i will fix.

**Comment by sugaarK** [comment]
> @sugaarK Now "squash and merge" is activated by default, so even 100 commits in the PR should be fine 👍

 Lols yeah @limonspb  would be very happy

**Comment by limonspb** [comment]
> > @sugaarK Now "squash and merge" is activated by default, so even 100 commits in the PR should be fine 👍
> 
> Lols yeah @limonspb would be very happy

heck yeah!

**Comment by KarateBrot** [comment]
@dengdizzy Looks like you also need to choose a line ending option. In Notepad++ for example you can go to `Edit -> EOL Conversion -> UNIX (LF)`.

Sorry for the inconvenience.

https://knowledge.digicert.com/solution/vmc-convert-cr-lf-to-lf-before-submitting-svg.html

**Comment by dengdizzy** [comment]
> @dengdizzy Looks like you also need to choose a line ending option. In Notepad++ for example you can go to `Edit -> EOL Conversion -> UNIX (LF)`.
> 
> After that you have UTF-8 encoding and only `\LF` line endings and it should finally work. Sorry for the inconvenience.
> 
> https://knowledge.digicert.com/solution/vmc-convert-cr-lf-to-lf-before-submitting-svg.html

Thank you for your hlpe,i will try it.

File: configs/default/TMTR-TMOTORVELOXF7V2.config [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Betaflight / TMOTORVELOXF7V2 (TMR7) 4.3.1 Sep 23 2022 / 17:19:06 () MSP API: 1.44
+
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_DPS310
```
Comment (haslinghuis) [inline configs/default/TMTR-TMOTORVELOXF7V2.config:5]: Please remove BARO - if not physical present on board.

File: configs/default/TMTR-TMOTORVELOXF7V2.config [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+#define USE_FLASH_M25P16
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
```
Comment (haslinghuis) [inline configs/default/TMTR-TMOTORVELOXF7V2.config:8]: Please remove MAG - if not physical present on board

File: configs/default/TMTR-TMOTORVELOXF7V2.config (hunk 1 above)
Comment (dengdizzy) [reply configs/default/TMTR-TMOTORVELOXF7V2.config:5]: FC has DPS310 on board.

File: configs/default/TMTR-TMOTORVELOXF7V2.config (hunk 2 above)
Comment (dengdizzy) [reply configs/default/TMTR-TMOTORVELOXF7V2.config]: I have removed it.

**Review by sugaarK — APPROVED** [review]

**Pull request merged**